### PR TITLE
Updated profiles, some changes to homepage

### PR DESCRIPTION
**Pull request by fraszle**
Updated profiles with queries, and re-factored. Starting to make changes to homepage, the Account tab now displays user_profile. Will update the Home tab to display a list of pets. Will need to figure out how to connect onClick of a pet to a pet profile page. Still need to work on pet image solutions

### CODE REVIEW

File: plentyofpets/lib/components/pet_description.dart [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      Text('Availability: ${data['availability']}'),
+      Text('Date Created: ${dateConverter(data['timestamp'])}'),
+      const Text('Disposition:'),
+      dispositionCol(data['disposition']),
+    ]);
+  }
+}
+
+// This data converter function takes a timestamp and coverts it to a
+// MM/DD/YY string
```
Comment (jmarr445) [inline plentyofpets/lib/components/pet_description.dart]: This function looks great, but this feels like it should be in a separate utility class so it's easily reusable

File: plentyofpets/lib/components/user_profile.dart [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            if (snapshot.connectionState == ConnectionState.done) {
+              Map<String, dynamic> data =
+                  snapshot.data!.data() as Map<String, dynamic>;
+
+              return Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text('Name: ${data['firstName']}' +
+                        ' ' +
+                        '${data['lastName']}'),
```
Comment (jmarr445) [inline plentyofpets/lib/components/user_profile.dart]: Do we need to use the template string here?  I kind of expected last name to be a string already. Same with the state on line 37

**Review by jmarr445 — CHANGES_REQUESTED** [review]

File: plentyofpets/lib/components/pet_description.dart (hunk 1 above)
Comment (fraszle) [reply plentyofpets/lib/components/pet_description.dart]: Great idea! I've pulled the function out into it's own file in the components folder, then referenced the file in the pet_description. I tested it, and it looks like it still works!

File: plentyofpets/lib/components/user_profile.dart (hunk 2 above)
Comment (fraszle) [reply plentyofpets/lib/components/user_profile.dart]: You're totally right, I've refactored and removed the template string for last name and state.

**Pull request merged**